### PR TITLE
OVS DPDK & SRIOV VA

### PR DESCRIPTION
**Pull request by Jaganathancse**
This change is to provide composed OVS DPDK & SRIOV deplpyment VA.

### CODE REVIEW

File: examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+            mtu: {{ min_viable_mtu }}
+            use_dhcp: false
+            dns_servers: {{ ctlplane_dns_nameservers }}
+            domain: {{ dns_search_domains }}
+            addresses:
+            - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+            routes: {{ ctlplane_host_routes }}
+            members:
+            - type: interface
+              name: eno2
```
Comment (abays) [inline examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:78]: Maybe since relative mappings are used below, we should use the relative name here too?

Comment (Jaganathancse) [reply examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:78]: @abays yes, will update it. thanks.

File: examples/va/nfv/ovs-dpdk-sriov/service-values.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      [ml2_type_vlan]
+      network_vlan_ranges = dpdk1:206:209,dpdk2:206:209,sriov1,sriov2 # CHANGE
+      [neutron]
+      physnets = dpdk1, dpdk2 # CHANGE
+      [neutron_physnet_dpdk1] # CHANGE
+      numa_nodes = 0 # CHANGE
+      [neutron_physnet_dpdk2 # CHANGE
+      numa_nodes = 0 # CHANGE
+      [neutron_tunnel] # CHANGE
+      numa_nodes = 0 # CHANGE
```
[the remark this answers is not in the document]
Comment (Jaganathancse) [reply examples/va/nfv/ovs-dpdk-sriov/service-values.yaml:34]: Ack

File: examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            - type: ovs_dpdk_port
+              name: dpdk2
+              members:
+              - type: interface
+                name: nic4
+          - type: sriov_pf
+            name: nic5
+            numvfs: 10
+            use_dhcp: false
+            promisc: true
```
Comment (mnietoji) [inline examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:118]: Set mtu 9000 for pf
             mtu: 9000

File: examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+          - type: ovs_user_bridge
+            name: br-link1
+            use_dhcp: false
+            ovs_extra: "set port br-link1 tag={{ lookup('vars', networks_lower['Tenant'] ~ '_vlan_id') }}"
+            addresses:
+            - ip_netmask: {{ lookup('vars', networks_lower['Tenant'] ~ '_ip') }}/{{ lookup('vars', networks_lower['Tenant'] ~ '_cidr') }}
+            mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+            members:
+            - type: ovs_dpdk_port
+              name: dpdk1
```
Comment (mnietoji) [inline examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:100]: Set number of queues for the port:
               rx_queue: 1

**Review by mnietoji — CHANGES_REQUESTED** [review]

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**